ndcube/tests/helpers.py

```diff
@@ -140,7 +143,13 @@ def assert_wcs_are_equal(wcs1, wcs2):
     assert wcs1.world_axis_units == wcs2.world_axis_units
     assert_equal(wcs1.axis_correlation_matrix, wcs2.axis_correlation_matrix)
     assert wcs1.pixel_bounds == wcs2.pixel_bounds
-
+    if wcs1.pixel_shape is not None:
+        random_idx = np.random.randint(wcs1.pixel_shape,size=[10,wcs1.pixel_n_dim])
+        # SlicedLowLevelWCS vs BaseHighLevelWCS don't have the same pixel_to_world method
+        low_level_wcs = wcs1
+        if isinstance(wcs1, BaseHighLevelWCS):
+            low_level_wcs = wcs1.low_level_wcs
+        np.testing.assert_array_equal(low_level_wcs.pixel_to_world_values(*random_idx.T), low_level_wcs.pixel_to_world_values(*random_idx.T))
```

Comment:
Is this line just testing that `low_level_wcs` is equivalent to itself?

Reply:
No, it was meant to test two different low level wcs from the input cubes. 

This is a typo.